plugins/woo-ai/src/product-description/product-description-button-container.tsx

```diff
@@ -277,15 +281,17 @@ export function WriteItForMeButtonContainer() {
 				) }
 			/>
 			{ shortDescriptionGenerated && (
-				<InfoModal
+				<TourSpotlight
 					id="shortDescriptionGenerated"
+					reference="#postexcerpt"
 					// message should be translatable.
-					message={ __(
+					description={ __(
 						'The short description was automatically generated by AI using the long description. This normally appears at the top of your product pages.',
 						'woocommerce'
 					) }
 					// title should also be translatable.
 					title={ __( 'Short Description Generated', 'woocommerce' ) }
+					placement="top"
```

Comment:
There's a regression introduced here after the refactoring; the spotlight shown after generating a short description will show up over and over, even if it's dismissed. I think now we have to pass the onDismiss() handler into the TourSpotlight component?

Reply:
Right! Good catch. I had to move the preference logic up a level and forgot about this one. I've just added it in https://github.com/woocommerce/woocommerce/pull/40692/commits/7ebf9033e15bfefe18b9911cad59823afe966efc